UNRELEASED.md

```diff
@@ -10,3 +10,11 @@
          * Another change description, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+### SANY importer
```

Comment:
I wonder if this shouldn't just be under `Features`?

Reply:
It should. I just followed the template in `UNRELEASED.md` :)

Reply:
If you think the example in `UNRELEASED.md` could be improved for clarity, a fix to the preamble in https://github.com/shonfeder/apalache/blob/shon/fix-release-of-zip/script/release-notes.sh#L19-L30 would be appreciated :)